### ISSUE TITLE
Attach behaviour of Charba charts in GWT

### ISSUE THREAD

**Issue by MartinSchwarzbauer** [pepstock-org/Charba#59]
Hi *!

I am using latest charba version (master - 4.0) and I have seen the following behaviour:

I open menu where i show a line chart (in a VerticalPanel - container)
Charts is shown and everything is good
Then i navigate to a new menu entry (chart is detached - i see this because i added the attach-handler)
when i go back to the menu entry where chart is shown (chart is attached again) but nothing is drawn???
** Chart is now gone :) Datasets etc are inside ... calling chart.update() or chart.draw() has no impact!
What can i do, that graph is redrawn?

Thank you very much,
Martin

**Comment by MartinSchwarzbauer** [comment]
After moving back to menu entry with chart: 
chart.isChartAttached()  returns false! How can i attach it again?

**Comment by stockiNail** [comment]
@MartinSchwarzbauer the normal behavior is destroy the chart when the panel container is detached.

But there is a property `destroyOnDetach` (default is `true`) that you can set which enables do not detach the chart. This property can be set on the options and directly to the chart instance.

```java
LineChart chart  = new LineChart();
chart.setDestroyOnDetach(false);
// or 
chart.getOptions().setDestroyOnDetach(false);
// or if this behavior is for all charts in the application
Defaults.get().getGlobal().setDestroyOnDetach(false);
```

Let me know if works.

**Comment by stockiNail** [comment]
@MartinSchwarzbauer it's documented here: https://pepstock-org.github.io/Charba-Wiki/docs/defaults/DefaultsCharts#commons-charts-options

**Comment by stockiNail** [comment]
> But I think for Widgets in GWT the behaviour is different !

@MartinSchwarzbauer yeah, it depends on the coder. For instance someone is creating the widget every time, someone like hiding the widgets. The default to `false` was thought in order to reduce the amount of memory consumption but allowing the user to decide the different behavior based on use case. 

In the Charba showcase for GWT, using `UiBinder` , we remove the widget and recreate every time is requested by the user.
